src/ui/DateRangePicker.js

```diff
@@ -129,6 +129,11 @@ function getInputRanges(inputRanges) {
   }));
 }
 
+function parseDate(date) {
+  const transformedDate = moment(date);
+  return (_.isNil(date) || !transformedDate.isValid()) ? undefined : transformedDate.toDate();
```

Comment:
I guess I would have done the `_.isNil(date)` before transforming the date, but not really important.